prometheus_pgbouncer_exporter/cli.py

```diff
@@ -31,6 +51,10 @@ def main():
 
     # Register signal handler
     def _on_sighup(signal, frame):
+        logging.getLogger().info("Received SIGHUP - Reloading pgbouncer hosts")
+        config = Config()
```

Comment:
use your function?

Reply:
lol, that's what I thought I did, but clearly skipped on.